lib/monaco-adapter.js

```diff
@@ -133,49 +160,7 @@ firepad.MonacoAdapter = (function () {
         this.monaco.executeEdits("my-source", [ops]);
       }
     }
-    return this.grabDocumentState();
-  };
-
-  // Index in monaco starts from 1
-  MonacoAdapter.prototype.posFromIndex = function (index, content) {
```

Comment:
Seems like `getPositionAt` is built-in and we had implemented our own.  Any reason why we did this in the past?  Is `getPositionAt` new?

Also the deleted comment here says the indexing in monaco starts at 1 but the monaco docs say `getPositionAt` is a 0-based index:
https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.itextmodel.html#getpositionat

Are we possibly introducing a new off-by-one error?

Reply:
@samtstern Monaco Index starts from one as it is a content string, array of chars. Position is formed as grid, (0,0) means you're at origin, i.e., cursor at the beginning of the document

Reply:
^^ exactly. @samtstern I can't comment on the historical choices as I'm not privy to them, but I believe Progyan's comment is correct. This API may not have been available at the time.